src/terms.js

```diff
@@ -49,6 +49,7 @@ export default [
     'billboard',
     'block.?chain',
     'blazing(ly)? fast',
+    'bleeding edge',
```

Comment:
To capture `bleeding-edge`:

```suggestion
    'bleeding.edge',
```